### PR TITLE
Support API 4.0.0

### PR DESCRIPTION
**Pull request by NhanAZ**
I have tested and the plugin functions work properly.
Please review the code... Thanks.

### CODE REVIEW

File: src/KygekDev/Utils/utils/DiskUsage.php [hunk 1]
```diff
@@ -5,17 +5,17 @@
...
-use pocketmine\command\PluginCommand;
+use pocketmine\command\Command;
 use pocketmine\plugin\Plugin;
 use pocketmine\Server;
 use pocketmine\utils\TextFormat as TF;
 use RecursiveDirectoryIterator as RDI;
 use RecursiveIteratorIterator as RII;
 
-class DiskUsage extends PluginCommand {
+class DiskUsage extends Command {
```
Comment (KygekDev) [inline src/KygekDev/Utils/utils/DiskUsage.php:15]: Could you also implement the `PluginOwned` interface?

File: src/KygekDev/Utils/utils/MemoryUsage.php [hunk 2]
```diff
@@ -5,14 +5,14 @@
 namespace KygekDev\Utils\utils;
 
 use pocketmine\command\CommandSender;
-use pocketmine\command\PluginCommand;
+use pocketmine\command\Command;
 use pocketmine\plugin\Plugin;
 use pocketmine\utils\TextFormat as TF;
 
-class MemoryUsage extends PluginCommand {
+class MemoryUsage extends Command {
```
Comment (KygekDev) [inline src/KygekDev/Utils/utils/MemoryUsage.php:12]: Same with this.

File: src/KygekDev/Utils/utils/DiskUsage.php (hunk 1 above)
Comment (NhanAZ) [reply src/KygekDev/Utils/utils/DiskUsage.php:15]: I don't know. Can you make a suggestion?
I found it working fine so I created a PR...

Comment (KygekDev) [reply src/KygekDev/Utils/utils/DiskUsage.php:15]: Here's an example:
https://github.com/KygekTeam/KygekRanksUI/blob/pm4/src/Kygekraqmak/KygekRanksUI/Commands.php

**Comment by NhanAZ** [comment]
Looks like I can't afford to upgrade this plugin yet. I surrender...
Thank u for considering PR!

**Comment by KygekDev** [comment]
> Looks like I can't afford to upgrade this plugin yet. I surrender... Thank u for considering PR!

@NhanAZ Why? Do you don't know how to implement `PluginOwned` in commands?

**Comment by NhanAZ** [comment]
That's right, I've never used this method to register commands
I usually register commands like this plugin
https://github.com/nhanaz-pm-pl/OpList
I was afraid to waste your time so I closed PR...
If possible, could you please update this plugin to API 4.0.0 soon? I can learn a few more things.

**Comment by KygekDev** [comment]
> That's right, I've never used this method to register commands
I usually register commands like this plugin
https://github.com/nhanaz-pm-pl/OpList

You used the classic method, that's fine for simple commands.

> I was afraid to waste your time so I closed PR...

No need to afraid wasting my time, unless you're doing something useless to me.

> If possible, could you please update this plugin to API 4.0.0 soon? I can learn a few more things.

I'll update when I have the motivation, but to be honest, this plugin isn't in my prioritized tasks list. The reason I uploaded this plugin to GitHub is for anyone to learn and modify. Anyway, this plugin shouldn't be used in servers other than mine, because this plugin is actually designed only for personal use.

**Pull request closed without merging**